cmd/api/main.go

```diff
@@ -134,25 +129,52 @@ func main() {
 	hs.SetServingStatus("tekton.results.v1alpha2.Results", healthpb.HealthCheckResponse_SERVING)
 	healthpb.RegisterHealthServer(s, hs)
 
-	// Prometheus metrics
-	grpc_prometheus.Register(s)
+	// Start prometheus metrics server
+	prometheus.Register(s)
 	http.Handle("/metrics", promhttp.Handler())
 	go func() {
-		if err := http.ListenAndServe(":8080", promhttp.Handler()); err != nil {
-			log.Fatalf("error running Prometheus HTTP handler: %v", err)
+		log.Printf("Prometheus server listening on: %s", configFile.PROMETHEUS_PORT)
+		if err := http.ListenAndServe(":"+configFile.PROMETHEUS_PORT, promhttp.Handler()); err != nil {
+			log.Fatalf("Error running Prometheus HTTP handler: %v", err)
 		}
 	}()
 
-	// Listen on port and serve.
-	port := os.Getenv("PORT")
-	if port == "" {
-		// Default gRPC server port to this value from tutorials (e.g., https://grpc.io/docs/guides/auth/#go)
-		port = "50051"
+	// Start gRPC server
+	lis, err := net.Listen("tcp", ":"+configFile.GRPC_PORT)
+	if err != nil {
+		log.Fatalf("Failed to listen: %v", err)
+	}
+	go func() {
+		log.Printf("gRPC server listening on: %s", configFile.GRPC_PORT)
+		log.Fatal(s.Serve(lis))
+	}()
+
+	// Load REST client TLS cert to connect to the gRPC server
+	if tlsError == nil {
+		creds, err = credentials.NewClientTLSFromFile(path.Join(configFile.TLS_PATH, "tls.crt"), configFile.TLS_OVERRIDE)
```

Comment:
Name update per my previous comment:
```suggestion
		creds, err = credentials.NewClientTLSFromFile(path.Join(configFile.TLS_PATH, "tls.crt"), configFile.TLS_HOSTNAME_OVERRIDE)
```